Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1051,7 +1047,8 @@ public IEnumerable<LocalPlugin> GetLocalPlugins(string path, bool cleanup = true
                     var metafile = Path.Combine(dir, "meta.json");
                     if (File.Exists(metafile))
                     {
-                        var manifest = _jsonSerializer.DeserializeFromFile<PluginManifest>(metafile);
+                        using FileStream jsonStream = File.OpenRead(metafile);
+                        var manifest = JsonSerializer.DeserializeAsync<PluginManifest>(jsonStream, jsonOptions).GetAwaiter().GetResult();
```

Comment:
Didn't know there was no sync overload that accepts a `Stream` :thinking:, in this case deserializing from a string/raw bytes is probably better (or at least cleaner)